### DISCUSSION TITLE
RealTime LivePaint, Inpaint and ImageToImage Preview Demo

### DISCUSSION THREAD

**Discussion by TheyCallMeHex** [saddam213/OnnxStack#42] [Show and tell]
https://www.youtube.com/watch?v=B07Gm4v17lA

A preview demo video of the real time features being added to OnnxStack in the upcoming release for real time Live Paint, real time Inpaint and real time ImageToImage diffusion.

**Comment by Amin456789** [comment]
nice! i just can't wait for this friday's release! lcm inpaiting, live paint...
